android/app/src/main/kotlin/com/jabook/app/jabook/compose/feature/player/PlayerScreen.kt

```
@@ -719,6 +908,138 @@ public fun PlayerScreen(
+            },
+        )
+    }
+}
+
+@Composable
+private fun StarRatingRow(
+    selected: Int,
+    onRate: (Int) -> Unit,
+) {
+    Row(horizontalArrangement = Arrangement.spacedBy(4.dp)) {
+        (1..5).forEach { star ->
```

detekt (1.23.8)

[warning] Using the forEach method on ranges has a heavy performance cost. Prefer using simple for loops.

(detekt.performance.ForEachOnRange)